### ISSUE TITLE
Define User Testing Tasks

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/pulfalight#237]
Goals of user testing:

1. Discover if we need advanced search. See #99

**Comment by jrgriffiniii** [comment]
A usability testing script should be drafted in collaboration with @axamei and @sdellis in order to ensure that there is a reference for guiding users through the interfaces.

**Comment by jrgriffiniii** [comment]
Regarding accessibility, it would be ideal if an unsighted person could be recruited for testing, and then could use only the keyboard for navigating the interfaces with a screen reader.

**Comment by jrgriffiniii** [comment]
Please see the `Usability testing script` in the shared Google Drive for an example usability testing script.

**Comment by hackmastera** [comment]
https://docs.google.com/document/d/1V2yPYpX-YjzZ6fUPrTyhBWlzYAIUNZE3P9EenpqPxNY/edit#heading=h.h1kbujq3z7xy is what we started writing up

**Comment by sdellis** [comment]
@hackartisan the document you posted is titled as a QA/Accessibility Script, but the title of this issue says it's a Usability Testing Script. Since these are two different things, we should be clear on the goals. Are we drafting a checklist for QA sign-off, or a list of tasks to observe users try to complete?

**Comment by hackartisan** [comment]
@sdellis Ah, I see now there is a discrepancy. I think what we need most is the QA / Accessibility script. I'm going to change the title of this issue. I don't know of any plans to do usability testing at this time but if there are some let's create another issue

**Comment by faithc** [comment]
We are planning to do usability testing as well.

**Comment by kelynch** [comment]
Yes, I agree that it makes sense to split out accessibility and usability testing into two separate sets of considerations.  We're prioritizing accessibility needs right now, and we can be more flexible for conducting usability testing, factoring in staff availability and when the ASpace/PULFAlight switchover will take place.
